src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementsByClassName('logo')[0];
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => resolve('Promise was resolved!'));
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1.then((success) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = success;
+  document.body.appendChild(div);
+});
+
+promise2.catch((error) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message', 'error-message');
+  div.textContent = error;
```

Comment:
When setting the `textContent` of the div in the `promise2.catch` block, you should use `error.message` instead of `error` to display the error message string. This is because `error` is an Error object, and you need to access its `message` property to get the string representation.